apps/apps.c

```diff
@@ -2695,5 +2695,5 @@ void make_uppercase(char *string)
     int i;
 
     for (i = 0; string[i] != '\0'; i++)
-        string[i] = toupper((unsigned char)string[i]);
+        string[i] = ossl_toupper((unsigned char)string[i]);
```

Comment:
This one is different from above, but I'd say we are still better off keeping it locale-specific.